packages/beacon-node/test/unit/api/impl/node/node.test.ts

```diff
@@ -52,15 +49,13 @@ describe("node api implementation", function () {
       const enr = SignableENR.createV4(keypair);
       enr.setLocationMultiaddr(multiaddr("/ip4/127.0.0.1/tcp/36001"));
       networkStub.getEnr.returns(Promise.resolve(enr));
-      networkStub.metadata = {
-        get json(): altair.Metadata {
-          return {
-            attnets: BitArray.fromBoolArray([true]),
-            syncnets: BitArray.fromBitLen(0),
-            seqNumber: BigInt(1),
-          };
-        },
-      } as MetadataController;
+      networkStub.getMetadata.returns(
```

Comment:
might just be able to do 

```suggestion
      networkStub.getMetadata.resolves(
```

and remove the `Promise.resolve` one line below